src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/EnumShouldNotHaveDuplicatedValues.cs

```diff
@@ -87,7 +87,7 @@ public sealed override void Initialize(AnalysisContext context)
                         .OfType<IFieldReferenceOperation>()
                         .Where(fro => fro.Field.ContainingType.Equals(enumSymbol))
                         .Any();
-                    if (onlyReferencesOneField)
+                    if (onlyReferencesOneField || fieldInitializerOperation.Value.ConstantValue.Value == null)
```

Comment:
Can this be moved the line above where we check `!fieldInitializerOperation.Value.ConstantValue.HasValue`

Reply:
Also tagging @Evangelink for review